author	Syam Sidhardhan <s.syam@samsung.com>
commit	d96457e1a4eefdf270332707518060490faf59a1
	Fri, 13 Apr 2012 09:35:51 +0000 (12:35 +0300)

health: Fix memory leak

diff -cr bluez-4.99.pan.orig/health/mcap.c bluez-4.99.pan/health/mcap.c
*** bluez-4.99.pan.orig/health/mcap.c	Wed Dec 21 17:53:54 2011
--- bluez-4.99.pan/health/mcap.c	Mon Oct 15 04:04:04 2012
***************
*** 901,906 ****
--- 901,907 ----
  		default:
  			g_set_error(err, MCAP_ERROR, MCAP_ERROR_INVALID_ARGS,
  						"Unknown option %d", cb);
+ 			g_free(c);
  			return FALSE;
  		}
  		cb = va_arg(args, int);
